bin/offline/online-download-images.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -ex
+
+#####################################################
+# PLEASE UPDATE VERSIONS BELOW!
+#####################################################
+
+BACKEND_VERSION=2.60.0
+WEBAPP_VERSION="42720-0.1.0-64e6cb-v0.22.0-production"
+TEAM_VERSION="10562-2.8.0-9e1e59-v0.22.1-production"
+ACCOUNT_VERSION="242-2.0.1-c4282e-v0.20.4-production"
+
+###########################################################
+# You should not need to change the code below
+############################################################
+
+# prepare destination folder
+NOW=$(date +"%Y-%m-%d_%H-%M-%S")
+FOLDER="update-$NOW"
+LOAD_SCRIPT="$FOLDER/load_into_registry.sh"
+mkdir "$FOLDER"
+
+# now that versions are known, write load_into_registry script
+echo "#!/usr/bin/env bash
+
+set -ex
+
+for image in *.tar; do
+    docker load < \"\$image\"
+done
```

Comment:
What are these tarballs?

Reply:
There are two scripts (the main script writes the second script):

the main script does `docker save quay.io/wire/brig:2.60.0 > brig.tar`

The second script (created by the main one) then does `docker load < brig.tar`

Does this need more comments to be clear?

Reply:
added comments in https://github.com/wireapp/wire-server-deploy/pull/128/commits/0825a1ffd8e096f34b57729df19816bffdb69177